src/scanpy/tools/_types.py

```diff
@@ -0,0 +1,9 @@
+from __future__ import annotations
+
+from typing import TypedDict
+
+
+class UmapMethodKwds(TypedDict, total=False):
+    dens_lambda: float
+    dens_frac: float
+    dens_var_shift: float
```

Comment:
This is `densmap` specific though?

Reply:
Would you prefer I rename it to `DensmapMethodKwds`?

Reply:
If the answer to my first question was "yes" (which I think it is, judging by the code), then yes.